apps/client/src/pages/ConclusionPage.js

```diff
@@ -86,6 +136,43 @@ const ConclusionPage = ({ topic, checker }) => {
           </Fragment>
         ))}
 
+        {!isMobile && (
+          <ComponentWrapper>
+            <PrintButton
+              type="button"
+              color="primary"
+              onClick={handlePrintButton}
+            >
+              Conclusie opslaan
+            </PrintButton>
+          </ComponentWrapper>
+        )}
+
+        <PrintOnly>
+          <Heading forwardedAs="h2" styleAs="h1">
+            Uw antwoorden
+          </Heading>
+          <Paragraph>
+            Hieronder kunt u per vraag uw gegeven antwoord teruglezen.
+          </Paragraph>
+          <QuestionAnswerTable checker={checker} />
+        </PrintOnly>
+
+        <PrintOnly style={{ marginTop: 20 }} withBorder avoidPageBreak>
+          <Alert
+            style={{
+              display: "block", // IE11 Fix
+            }}
+            heading="Let op"
+            content="De vergunningcheck is nog in ontwikkeling. Hierdoor kunnen wij nog
+              geen zekerheid bieden dat de uitkomst correct is. Ook is de
+              informatie nog niet voor iedereen goed te lezen of te beluisteren.
+              Wilt u iets zeker weten of wilt u meer informatie? Bel het
+              telefoonnummer 14 020, maandag tot en met vrijdag van 08.00 uur
+              tot 18.00 uur."
```

Comment:
Consider using a backtick for multiline strings.

Reply:
Consider it done